app/src/main/java/com/matmoongi/screens/LoginScreen.kt

```diff
@@ -20,7 +21,7 @@ import androidx.compose.ui.unit.dp
 import com.matmoongi.R
 
 @Composable
-fun LoginScreen(onClickSkipLoginButton: () -> Unit) {
+fun LoginScreen(onClickSkipLoginButton: () -> Unit, onClickNaverLoginButton: () -> Unit) {
```

Comment:
[nitpick]
```suggestion
fun LoginScreen(
    onClickSkipLoginButton: () -> Unit,
    onClickNaverLoginButton: () -> Unit
) {
```

이렇게 줄바꿈을 하는 편이 저에겐 약간 읽기가 쉬운듯 하네요.

Reply:
넵! 개선하겠습니다